### PR TITLE
add sort_categories argument

### PR DESCRIPTION
**Pull request by chris-b1**

### CODE REVIEW

File: pandas/types/concat.py [hunk 1]
```diff
@@ -211,22 +211,23 @@ def convert_categorical(x):
...
     Combine list-like of Categoricals, unioning categories. All
-    must have the same dtype, and none can be ordered.
+    categories must have the same dtype.
 
     .. versionadded:: 0.19.0
 
     Parameters
     ----------
     to_union : list-like of Categoricals
+    sort_categories : boolean, default False
```
Comment (jreback) [inline pandas/types/concat.py:224]: I would just call this `sort`

Comment (chris-b1) [inline pandas/types/concat.py:224]: Isn't that kind of ambiguous as to whether it will sort the categories or the values in the resulting Categorical?

Comment (sinhrks) [inline pandas/types/concat.py:224]: I also prefer `sort_categories` to be explicit. 

BTW, I recommend @chris-b1 to open a separate PR on pandas master for open discussion.

Comment (chris-b1) [inline pandas/types/concat.py:224]: Sure once you merge https://github.com/pydata/pandas/pull/13763, I can submit as a standalone PR

**Pull request closed without merging**